python/test/test_data.py

```diff
@@ -254,3 +254,72 @@ def test_brackets(self):
 
         # passing a range that is completely out of bounds throws a KeyError
         assert_raises(KeyError, self.series.__getitem__, (slice(2, 3), slice(None, None)))
+
+class TestDataMethods(PySparkTestCase):
+
+    def test_sortbykey(self):
+
+        dataLocal = [
+            ((0, 0), array([0])),
+            ((0, 1), array([0])),
+            ((0, 2), array([0])),
+            ((1, 0), array([0])),
+            ((1, 1), array([0])),
+            ((1, 2), array([0]))
+        ]
+
+        data = Data(self.sc.parallelize(dataLocal))
+        out = data.sortByKey().keys().collect()
+        assert(array_equal(out, [(0, 0), (1, 0), (0, 1), (1, 1), (0, 2), (1, 2)]))
+
+        dataLocal = [
+            ((0,), array([0])),
+            ((1,), array([0])),
+            ((2,), array([0]))
+        ]
+
+        data = Data(self.sc.parallelize(dataLocal))
+        out = data.sortByKey().keys().collect()
+        assert(array_equal(out, [(0,), (1,), (2,)]))
+
+    def collect(self):
+
+        dataLocal = [
+            ((0, 0), array([0])),
+            ((0, 1), array([1])),
+            ((0, 2), array([2])),
+            ((1, 0), array([3])),
+            ((1, 1), array([4])),
+            ((1, 2), array([5]))
+        ]
+
+        data = Data(self.sc.parallelize(dataLocal))
+
+        out = data.collectKeysAsArray()
+
+        assert(array_equal(out, [[0, 0], [0, 1], [0, 2], [1, 0], [1, 1], [1, 2]]))
+
+        out = data.collectKeysAsArray()
```

Comment:
I might be missing something, but I think this should be
```
out = data.collectValuesAsArray()
```
Not sure how it's passing the unit tests at the moment though....